OpenSim/Tools/Test/sphere.vtp

```diff
@@ -0,0 +1,883 @@
+<?xml version="1.0"?>
+<VTKFile type="PolyData" version="0.1" byte_order="LittleEndian" compressor="vtkZLibDataCompressor">
```

Comment:
Great point regarding the sphere being invariant to rotations,  but since we haven't had non-mesh objects pre 4.0, whatever model that exercises conversion will have to use a mesh file. I can make one that's a sparse Mesh (since we don't have one off-the-shelf) if you think that strengthens the test. Please let me know.